packages/client/lib/CredentialRequestClientBuilder.ts

```diff
@@ -1,14 +1,14 @@
-import { CredentialFormat } from '@sphereon/ssi-types';
-
-import { CredentialRequestClient } from './CredentialRequestClient';
-import { convertURIToJsonObject } from './functions';
 import {
   AccessTokenResponse,
   EndpointMetadata,
   IssuanceInitiationRequestPayload,
   IssuanceInitiationWithBaseUrl,
   OpenID4VCIServerMetadata,
-} from './types';
+} from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.